src/coreclr/inc/stdmacros.h

```diff
@@ -159,9 +159,11 @@
     #define DBG_ADDR(ptr)      (DWORD)((UINT_PTR)(ptr))
 #endif // HOST_64BIT
 
-#ifdef TARGET_ARM
+#ifndef ALIGN_ACCESS
```

Comment:
Thank you. I updated.
For  `gcinfo_universal_arm` build with riscv64 target, `HOST_RISCV64` and `TARGET_ARM` is defined. So `LOG2_PTRSIZE` is not same. I just removed `gcinfo_universal_arm` and `gcinfo_win_x86` from `gcinfo/CMakeLists.txt` like I did in `jit/CMakeLists.txt`
IMO, `ALIGN_ACCESS` should be splitted for `HOST` and `TARGET`. I think `LOG2_PTRSIZE` and `ALIGN_ACCESS` in `pal_endian.h` only updated according to `HOST` like `LOG2_PTRISZE_HOST` and `ALIGN_ACCESS_HOST`. Actually, I am afraid of intermittent test failures or something. So I just updated the cmake file. What do you think?

Reply:
> ALIGN_ACCESS should be splitted for HOST and TARGET

`ALIGN_ACCESS` is always used in the HOST context currently. I agree that it would need to be split if we have cases where it needs to be used in the TARGET context.

Reply:
Thank you!
`LOG2_PTRSIZE` is defined in the HOST and TARGET context.
Is it also defined in HOST context only?
https://github.com/dotnet/runtime/blob/1949bd2e017a0dc1e48b7cfed263051a6ed147a5/src/coreclr/inc/stdmacros.h#L142-L146
https://github.com/dotnet/runtime/blob/1949bd2e017a0dc1e48b7cfed263051a6ed147a5/src/coreclr/pal/inc/pal_endian.h#L100-L108

Reply:
Yes, this can use some cleanup if it is causing problems.

Reply:
Thank you!